aspnetcore/signalr/supported-platforms.md

```diff
@@ -29,7 +29,7 @@ The [JavaScript client](https://www.npmjs.com/package/@aspnet/signalr) runs on N
  
 ## .NET client
 
-The [.NET client](https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR/) runs on any server platform supported by ASP.NET Core.
+The [.NET client](https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR/) can be used by developers using .NET Core. [Xamarin developers can use SignalR](https://github.com/aspnet/Announcements/issues/305) for building applications for Android using Xamarin.Android 8.4.0.1 and later, and for iOS using Xamarin.Android 11.14.0.4 and later. 
```

Comment:
What about ASP.NET Core projects targeting .NET Framework? I thought this Microsoft.AspNetCore.SignalR package worked there too.

Reply:
You could retain the original "runs on any platform supported by .NET Core" by just dropping "server" and "ASP" before .NET.  Then put "For example" in front of your Xamarin sentence.  Then I don't think you need to mention explicitly the ability of ASP.NET Core to target Framework, as that platform would be Windows, which is covered by "any platform supported by .NET Core".

Reply:
Shouldn't that read as "and for iOS using **Xamarin.iOS** 11.14.0.4 and later" ?

Reply:
Absolutely. Thanks for the catch @sfarmer-MesaAZ